src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  padding: $calendar-padding;
+
+  &__container {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: $week-size;
+  gap: $gap-between-days;
+  padding: $calendar-padding;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  @each $day, $value in $days {
+    &--start-day--#{$day} :nth-child(1) {
+      margin-left: $value * $width-gap-and-cell;
+    }
+  }
+
+  }
+
+  @for $index from 28 through 31 {
+    &--month-length--#{$index} {
+      :nth-child(n + #{$index}) {
+        display: none;
+      }
+    }
+  }
```

Comment:
this part of code doesn't work, when i change month length in dev tools, days amount still 28, fix it please